core/src/test/java/org/elasticsearch/action/admin/cluster/stats/ClusterStatsIT.java

```diff
@@ -181,6 +181,12 @@ public void testValuesSmokeScreen() throws IOException, ExecutionException, Inte
         assertThat(msg, response.nodesStats.getProcess().getMinOpenFileDescriptors(), Matchers.greaterThanOrEqualTo(-1L));
         assertThat(msg, response.nodesStats.getProcess().getMaxOpenFileDescriptors(), Matchers.greaterThanOrEqualTo(-1L));
 
+        assertThat(msg, response.nodesStats.getOs().getMem().getFree().bytes(), Matchers.greaterThanOrEqualTo(-1L));
+        assertThat(msg, response.nodesStats.getOs().getMem().getTotal().bytes(), Matchers.greaterThanOrEqualTo(-1L));
+        assertThat(msg, response.nodesStats.getOs().getMem().getUsed().bytes(), Matchers.greaterThanOrEqualTo(-1L));
```

Comment:
I think it should be `greaterThanOrEqualTo(0)`... According to https://docs.oracle.com/javase/8/docs/jre/api/management/extension/com/sun/management/OperatingSystemMXBean.html#getTotalPhysicalMemorySize-- both `getTotalPhysicalMemorySize` and `getFreePhysicalMemorySize` should return a value.